cmd/juju/action/run.go

```diff
@@ -61,9 +61,15 @@ type runCommand struct {
 }
 
 const runDoc = `
-Run a charm action for execution on a given unit, with a given set of params.
+Run a charm action for execution on the given unit(s), with a given set of params.
 An ID is returned for use with 'juju show-operation <ID>'.
 
+A action executed on a given unit becomes a task with an ID that can be
+used with 'juju show-task <ID>'.
```

Comment:
Maybe "becomes a task with an ID. You can check the task's status with 'juju show-task <ID>'."? I found the current wording a bit confusing.

Reply:
So the command behaves differently running an action on one unit than running it on multiple units (no operation created)? Is that going to be confusing? Is it only if you specifically target it at one unit? Or does the same thing happen if you specify a whole application but it only has one unit?

Oh - reading ahead I think I've gone off half-cocked - sounds like it always makes an operation and gives you the op-id, but if it only contains one task it also tells you that id for convenience. Fine then!

Reply:
correct :-)